oqamaint/update.py

```diff
@@ -56,15 +56,14 @@ def get_max_revision(self, job):
     def settings(self, src_prj, dst_prj):
         s = self._settings.copy()
 
+        patch_id = src_prj.split(':')[-1]
```

Comment:
this is equal to `def patch_id(repo):`? In INCIDENT patch is needed a real name of patch of which is incident number only small part

Reply:
hmm, not for openSUSE. that's what fooled me.

The actual problem is that https://download.opensuse.org/repositories/openSUSE:/Maintenance:/8006/openSUSE_Leap_15.0_Update/repodata/ does not have an updateinfo.xml - which might be a bug.

Reply:
CC @andreasstieger

Reply:
So I checked and you are right - my patch only works for openSUSE.

And http://download.opensuse.org/repositories/openSUSE:/Maintenance:/8027/openSUSE_Leap_15.0_Update/repodata/ also doesn't have updateinfo, but it must have had at some point.